### DISCUSSION TITLE
Checkpoints for Evasion attackers

### DISCUSSION THREAD

**Discussion by le000043** [Trusted-AI/adversarial-robustness-toolbox#1500] [Q&A] [answered]
Hello,
I am running experiments involving evasion attacks such as FGM or PGD on a cluster. The cluster only allows jobs to run 4-5 hours before they are requeued. Therefore, my experiments need to create checkpoints along the way. I can save the adversarial examples and predictions but it seemed the attackers are reset anew every time I load checkpoints. Are there any ways to get around this?
Thank you very much

**Comment by beat-buesser** [comment, marked as answer]
Hi @le000043 That's an interesting question. Are you planning to restart a PGD attack and run the remaining iterations until `max_iter` in a second job on the cluster?

**Reply by le000043** [reply]
yes, that was the plan. I had resolved it. The issue was found in my saving checkpoints algorithm.
